template/blt/project.yml

```diff
@@ -12,6 +12,11 @@ project:
   local:
     protocol: http
     hostname: local.${project.machine_name}.com
+  # These values will be used to generate the theme for this project.
+  subtheme:
```

Comment:
Unless there's a compelling reason to have a human name and a description, I think using simply `subtheme: cogsub` should suffice. However, I might default it to something dynamic like `${project.machine_name}_sub`.

To be honest , project.human_name is fairly useless right now.